front/src/config/i18n/en.json

```diff
@@ -245,6 +246,29 @@
         "noImageToShow": "No image to show",
         "editBoxNameLabel": "Enter the name you want to give to the box:",
         "editBoxNamePlaceholder": "Name of the box"
+      },
+      "chart": {
+        "defaultInterval": "Default interval",
+        "lastHour": "Last hour",
+        "lastDay": "Last 24 hours",
+        "lastSevenDays": "Last 7 days",
+        "lastThirtyDays": "Last 30 days",
+        "lastThreeMonths": "Last 3 months",
+        "lastYear": "Last year",
+        "noValue": "No values recorded on this interval",
+        "editNameLabel": "Enter the name of this box.",
+        "editDeviceFeaturesLabel": "Select the device you want to display here.",
```

Comment:
Ended with dot, but not other translations, is this expected?
(same on upper line)